spec/property_spec.rb

```diff
@@ -1,16 +1,27 @@
+    let!(:create_params) { { 'realestates.apartmentBuy': { 'externalId': 'extID123', 'title': 'RestAPi appartmentBuy', 'creationDate': '2014-08-12T15:56:52.000+02:00', 'address': { 'street': 'Andreasstraße', 'houseNumber': '10', 'postcode': '10243', 'city': 'Berlin', 'wgs84Coordinate': { 'latitude': 52.51245, 'longitude': 13.43134 } }, 'apiSearchData': { 'searchField1': 'apiSearchField1', 'searchField2': 'apiSearchField2', 'searchField3': 'apiSearchField3' }, 'realEstateState': 'INACTIVE', 'descriptionNote': 'Objektbeschreibung(noch 2000 Zeichen) description-note \n after line break this text is in the next row', 'furnishingNote': 'Ausstattung(noch 2000 Zeichen)', 'locationNote': 'Lage(noch 2000 Zeichen)', 'otherNote': 'Sonstiges(noch 2000 Zeichen)', 'showAddress': 'true', 'apartmentType': 'MAISONETTE', 'floor': 2, 'lift': 'true', 'energyCertificate': { 'energyCertificateAvailability': 'AVAILABLE', 'energyCertificateCreationDate': 'BEFORE_01_MAY_2014' }, 'cellar': 'YES', 'handicappedAccessible': 'YES', 'numberOfParkingSpaces': 1, 'condition': 'WELL_KEPT', 'lastRefurbishment': 2010, 'interiorQuality': 'SOPHISTICATED', 'constructionYear': 1995, 'freeFrom': 'sofort', 'heatingTypeEnev2014': 'FLOOR_HEATING', 'energySourcesEnev2014': { 'energySourceEnev2014': ['WOOD_CHIPS'] }, 'buildingEnergyRatingType': 'ENERGY_CONSUMPTION', 'thermalCharacteristic': 123.85, 'energyConsumptionContainsWarmWater': 'YES', 'numberOfFloors': 5, 'usableFloorSpace': 70, 'numberOfBedRooms': 1, 'numberOfBathRooms': 2, 'guestToilet': 'YES', 'parkingSpaceType': 'UNDERGROUND_GARAGE', 'rented': 'YES', 'rentalIncome': 895, 'listed': 'YES', 'parkingSpacePrice': 15_000, 'summerResidencePractical': 'YES', 'price': { 'value': 175_000, 'currency': 'EUR', 'marketingType': 'PURCHASE', 'priceIntervalType': 'ONE_TIME_CHARGE' }, 'livingSpace': 75, 'numberOfRooms': 3, 'energyPerformanceCertificate': 'true', 'builtInKitchen': 'true', 'balcony': 'true', 'certificateOfEligibilityNeeded': 'false', 'garden': 'true', 'courtage': { 'hasCourtage': 'YES', 'courtage': 'Provisionshöhe (brutto)', 'courtageNote': 'Provisionshinweis (noch 500 Zeichen)' }, 'serviceCharge': 575 } } }
```

Comment:
whoa, this is a single line! Not really want to nitpick, but wouldn't it be a bit easier to read if we format it properly?
(we can also use `irb` and `pp` and just copy-paste output from there)